vars/failOnServerError.groovy

```diff
@@ -0,0 +1,8 @@
+/**
+ * grep for errors in server logs throwing an abort exception
+ */
+def call(String path, String pattern = '^[0-9]{4}-[0-9]{2}-[0-9]{2}.*ERROR.*') {
```

Comment:
path => paths ?
paths: space-separated list of paths (files only? not recursive?) where the given pattern must not be found
paths: what is the format? ant-like as in the Text finder Plugin? does "**" works? 
pattern: defaults to match any ERROR line in the Nuxeo server logs